README.md

```diff
@@ -50,8 +50,21 @@ graphite.send(name, res, timestamp)
 ```
 
 ## Usage examples
-`graphite.add_sec_metric('delete_rps_max', function() return box.stat().DELETE.rps end, graphite.max)`
+``` lua
+graphite.add_sec_metric('delete_rps_max', function() return box.stat().DELETE.rps end, graphite.max)
+
+graphite.add('requests', 1)
+
+graphite.max_per_min('max_mysql_query_time', query_time)
 
-`graphite.add('requests', 1)`
+-- graphite.wrap_func adds for function dostuff following stats with one line:
```

Comment:
It's just I'm using this set. Do you want to add anything?

### Utility functions
``` lua
-- handy wrapper to add func_name_avg, func_name_min, func_name_max, func_name_rpm metrics to your function with a single line of code
graphite.wrap_func_with_stat(name, func)
```

## Usage examples
local function dostuff()
end

dostuff = graphite.wrap_func('dostuff', dostuff)
```